content/blog/2021-05-04-prometheus-conformance-remote-write-compliance.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Prometheus conformance program: Remote Write Compliance test results"
+created_at: 2021-05-05
+kind: article
+author_name: Richard "RichiH" Hartmann
+---
+
+As [announced by CNCF](https://www.cncf.io/blog/2021/05/03/announcing-the-intent-to-form-the-prometheus-conformance-program/) and by [ourselves](https://prometheus.io/blog/2021/05/03/introducing-prometheus-conformance-program/), we're starting a Prometheus conformance program.
+
+To give everyone an overview of where the ecosystem is before running tests officially, we wanted to show off the newest addition to our happy little bunch of test suites: The Prometheus Remote Write tests.
```

Comment:
Add a short sentence at the end of this paragraph explaining what Remote Write is?

Reply:
Also, that this is testing only the sending end, not the receiving end?

Reply:
doine